.env.gold

```
@@ -3,3 +3,21 @@ NEXT_PUBLIC_NEUTRON_TICKER=GAU
 NEXT_PUBLIC_PROTON_TICKER=GAUC
 NEXT_PUBLIC_PEG_ASSET=Gold
 NEXT_PUBLIC_PEG_UNIT=1g
+NEXT_PUBLIC_FAVICON=/logo/gluon.png
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_FAVICON key should go before the NEXT_PUBLIC_NEUTRON_TICKER key

(UnorderedKey)